packages/rps/src/redux/store.ts

```diff
@@ -24,28 +24,32 @@ const store = createStore(reducer, enhancers);
 function* rootSaga() {
   yield fork(loginSaga);
 
-  const channelProviderEnabled = yield stateChannelWalletSaga();
-
-  if (channelProviderEnabled) {
-    const client = new RPSChannelClient(new ChannelClient(window.channelProvider));
-    yield fork(channelUpdatedListener, client);
+  let client;
+  if (process.env.AUTO_OPPONENT === 'A' || process.env.AUTO_OPPONENT === 'B') {
```

Comment:
Would this work?
```suggestion
  if (process.env.AUTO_OPPONENT) {
```

Reply:
I think it would, but I actually prefer the current way in case in future someone wants to set `AUTO_OPPONENT=none` or any other unexpected value.